### PR TITLE
feat(ELB): fix elb service api comments

### PR DESCRIPTION
**Pull request by SabreKeyZ**
**What this PR does / why we need it**:
fix elb service api comments
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
fix elb service api comments
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
none
```

### CODE REVIEW

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go [hunk 1]
```diff
@@ -31,6 +31,10 @@ import (
 // @API ELB POST /v2.0/{project_id}/loadbalancers/{id}/tags/action
 // @API ELB GET /v2.0/{project_id}/loadbalancers/{id}/tags
 // @API EIP DELETE /v1/{project_id}/publicips/{id}
+// @API BSS GET /v2/orders/customer-orders/details/{order_id}
+// @API BSS POST /v2/orders/suscriptions/resources/query
+// @API BSS POST /v2/orders/subscriptions/resources/autorenew/{instance_id}
+// @API BSS DELETE /v2/orders/subscriptions/resources/autorenew/{instance_id}
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go:38]:
```suggestion
// @API BSS DELETE /v2/orders/subscriptions/resources/autorenew/{instance_id}
// @API BSS POST /v2/orders/subscriptions/resources/unsubscribe
```

Comment (SabreKeyZ) [reply huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go:38]: done

**Pull request merged**